src/Chapter1.hs

```diff
@@ -619,8 +631,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = let m = abs n
+                 lastD = m `mod` 10
+                 secondLastD = (m `div` 10) `mod` 10
```

Comment:
That is awesome!

You can see that you have both:

```haskell
m `mod` 10
```
and 
```haskell
m `div` 10
```

The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!

So you could write it this way:

```haskell
(d, lastD) = divMod m 10
```

You can see how we could pattern match on the pair 🙂